### PR TITLE
#932: fixed column drag and drop when some columns are hidden

### PR DESCRIPTION
**Pull request by ryanintulsa**
## Related Issue
https://github.com/mbrn/material-table/issues/932

## Description
Drag and drop column reordering was not considering hidden columns.

## Related PRs
none

## Impacted Areas in Application
data-manager

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,5 @@
 .docz
 .vscode
-dist
```
Comment (mbrn) [inline .gitignore]: Why did you removed this line? This makes your pr unreadable.

**Comment by ryanintulsa** [comment]
@mbrn sorry, an oversight. Please look again. If you're curious why this happens, it's because when you aren't ready to publish a fork to npm you can still reference it in github directly in package.json like "material-table": "git://github.com/ryanintulsa/material-table.git#myBranchName", but for that to work you must check in dist. It's confusing hopping back and forth, and I wish git had a better solution.

**Comment by ryanintulsa** [comment]
I've got a boundary condition error. I'm working on the fix.

**Comment by austinterris** [comment]
This is still an issue on 1.56.1.  It seems to not work the first time you try dragging over a hidden column but if you do the same action again then it will work.

**Pull request merged**